pkg/kubecfg/param.go

```diff
@@ -193,53 +194,23 @@ func outputParamsFor(component string, params param.Params, out io.Writer) error
 func outputParams(params map[string]param.Params, out io.Writer) error {
 	keys := sortedKeys(params)
 
-	//
-	// Format each component parameter information for pretty printing.
-	// Each component will be outputted alphabetically like the following:
-	//
-	//   COMPONENT PARAM     VALUE
-	//   bar       name      "bar"
-	//   bar       replicas  2
-	//   foo       name      "foo"
-	//   foo       replicas  1
-	//
-
-	maxComponentLen := len(paramComponentHeader)
-	for _, k := range keys {
-		if l := len(k); l > maxComponentLen {
-			maxComponentLen = l
-		}
+	rows := [][]string{
+		[]string{"COMPONENT", "PARAM", "VALUE"},
```

Comment:
There's constants on line 110 `paramComponentHeader`, etc. Can you use those instead?